package.json

```diff
@@ -1,48 +1,68 @@
 {
-  "name": "typescript-action",
-  "version": "0.0.0",
+  "name": "xealth-cherry-pick-action",
+  "version": "1.0.0",
   "private": true,
-  "description": "TypeScript template action",
-  "main": "lib/main.js",
+  "description": "A Cherry Pick GitHub Action 🍒",
+  "main": "dist/main.js",
   "scripts": {
     "build": "tsc",
     "format": "prettier --write **/*.ts",
     "format-check": "prettier --check **/*.ts",
     "lint": "eslint src/**/*.ts",
-    "package": "ncc build --source-map --license licenses.txt",
-    "test": "jest",
-    "all": "npm run build && npm run format && npm run lint && npm run package"
+    "package": "ncc build --source-map --license license.txt",
+    "test": "jest --runInBand",
+    "all": "npm run test && npm run build && npm run format && npm run lint && npm run package"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/actions/typescript-action.git"
+    "url": "git+https://github.com/Xealth/cherry-pick-action.git"
   },
   "keywords": [
     "actions",
-    "node",
-    "setup"
+    "cherry-pick",
+    "auto",
+    "automation",
+    "release",
+    "branch"
+  ],
+  "author": "carloscastrojumo (https://github.com/carloscastrojumo)",
```

Comment:
Remove author and contributors.

Remove or trim any fields such a this that are not required or serving a purpose.

Most of this file is only useful for npm libs which this is not.

So keep it simple to reduce maintenance drag.